src/Symfony/Component/Validator/Constraints/EmailValidator.php

```diff
@@ -65,7 +65,7 @@ public function validate($value, Constraint $constraint)
 
             $strictValidator = new \Egulias\EmailValidator\EmailValidator();
 
```

Comment:
The intent here is to send the 3rd argument as `false`, rather than `true`.  But thanks to the default values for arguments 2 and 3 (false and false, respectively), we can simply omit them completely.